src/rust/bitbox02-rust-c/src/lib.rs

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#![cfg_attr(not(test), no_std)]
+#![no_std]
 #![feature(alloc_error_handler)] // used in alloc.rs
```

Comment:
so, allow error handler is now also used in tests, right? just checking.

Reply:
No, those are both independent module level declarations. It means this crate is no_std, and the alloc_error_handler feature is used.